### PR TITLE
BZ1953350: Default OCPv provider (beta)

### PR DESCRIPTION
**Pull request by apinnick**
https://bugzilla.redhat.com/show_bug.cgi?id=1953350

beta and 2.0

Change previews:

https://deploy-preview-111--forklift-documentation.netlify.app/documentation/doc-migration_toolkit_for_virtualization/master/#adding-virt-provider_forklift (Adding virt provider)

https://deploy-preview-111--forklift-documentation.netlify.app/documentation/doc-migration_toolkit_for_virtualization/master/#creating-migration-plan_forklift (Migration plan prerequisites)

### CODE REVIEW

**Comment by apinnick** [comment]
@istein1  Ready for QE review.

File: documentation/doc-Migration_Toolkit_for_Virtualization/modules/creating-migration-plan.adoc [hunk 1]
```diff
@@ -12,15 +12,15 @@ A migration plan allows you to group virtual machines that should be migrated to
 .Prerequisites
 
 * You must create a VDDK image and add it to the `v2v-vmware` config map.
-* You must add source and target providers to the web console.
```
Comment (istein1) [inline documentation/doc-Migration_Toolkit_for_Virtualization/modules/creating-migration-plan.adoc:15]: I think here we can elaborate that the current target provider is added by default during MTV installation, and if the target provider is not the one from which we run the MTV migration, then this other target provider should be added.
Suggested text:
"If the target provider is the current cluster, then it's added duting MTV installation automatically. However if the target provider is another cluster, then it must be added"

Comment (apinnick) [reply documentation/doc-Migration_Toolkit_for_Virtualization/modules/creating-migration-plan.adoc:15]: Done

**Comment by istein1** [comment]
LGTM

**Pull request merged**